index.html

```diff
@@ -905,7 +907,59 @@ <h3>Join files together</h3>
       <p>For more information, see the <a href="https://trac.ffmpeg.org/wiki/Concatenate" target="_blank">FFmpeg wiki page on concatenating files</a>.</p>
       <p class="link"></p>
     </div>
-    <!-- ends Join files together -->
+    <!-- ends Join files of the same type together -->
+
+    <!-- Join files of different types together -->
+    <label class="recipe" for="join_different_files">Join (concatenate) two or more files of different types</label>
+     <input type="checkbox" id="join_different_files">
+     <div class="hiding">
+      <h3>Join files together</h3>
+      <p><code>ffmpeg -i input1.avi -i input2.mp4 -filter_complex "[0:v:0][0:a:0][1:v:0][1:a:0]concat=n=2:v=1:a=1[videoOut][audioOut]" -map "[videoOut]" -map "[audioOut]" <i>output_file</i></code></p>
+      <p>
+        This command takes two or more files of the different file types and joins them together to make a single file.<br>
+        However, it only works properly if the files to be combined have the same dimensions (e.g., 720x576). Some aspects of the input files will be normalised: for example, if an input file contains a video track and an audio track that do not have exactly the same duration, the shorter one will be padded. However, note that if the input files have different framerates, the output file will be of variable framerate.
+      </p>
```

Comment:
I would not change our coding style. Don’t add a newline after the `<p>` opening tag nor before the `</p>` closing tag.